strr-examiner-web/app/stores/examiner.ts

```diff
@@ -17,6 +19,25 @@ export const useExaminerStore = defineStore('strr/examiner-store', () => {
   const activeHeader = computed(() => {
     return activeRecord.value?.header
   })
+  const nocContent = reactive({
+    content: ''
+  })
+  const showNocModal = ref(true)
+  const nocFormRef = ref()
+
+  const sendNocSchema = computed(() => z.object({
+    content: z.string().min(1, { message: t('validation.nocContent') })
+  }))
+
+  const validateNocContent = async (): Promise<boolean> => {
```

Comment:
There's probably no need for this, there is a util method in base layer `validateForm`, which could be used in `validateFn`.

Reply:
Now relying on `validateForm`.